### ISSUE TITLE
Insufficient contrast for regular white text on blue background

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/coop-library#224]
**Describe the bug**
The regular white text on the blue background and background SVG artwork on Directory home page does not meet WCAG AAA contrast guidelines (1.4.6).

**To Reproduce**
1. Go to the Directory home page.
2. Use a contrast analyser and inspect the white text on the blue background (blue-400), and the blue background artwork.

**Screenshots**

Insufficient contrast for white text on artwork:
![image](https://user-images.githubusercontent.com/564762/76534290-ae702300-644f-11ea-8a1e-4ba1171478d9.png)

Insufficient contrast for white text on blue-400 background:
![image](https://user-images.githubusercontent.com/564762/76534244-9e584380-644f-11ea-990d-237944d07ebf.png)

**Comment by dayotte** [comment]
Let's do the following:
- upper header (PC Resource Library) bar: Dark Blue 1 (1C4342), with 90% opacity 
- lower header (Browse by) bar: Blue 500 (16605D) 
- all text in nav bar and header bars: change from offwhite (F9F9F7) to white (FFFFFF)
- @greatislander @cherylhjli I decided to leave the nav bar background colour as-is so it still works well with the other pages/backgrounds
